benchmarks/cugraph/pytest-based/bench_cugraph_uniform_neighbor_sample.py

```diff
@@ -100,12 +103,16 @@ def create_mg_graph(graph_data):
     Create a graph instance based on the data to be loaded/generated.
     """
     (client, cluster) = start_dask_client(
-        enable_tcp_over_ucx=False,
-        enable_infiniband=False,
-        enable_nvlink=False,
-        enable_rdmacm=False,
-        net_devices=None,
+        # enable_tcp_over_ucx=True,
+        # enable_infiniband=False,
+        # enable_nvlink=True,
+        # enable_rdmacm=False,
+        protocol="ucx",
+        rmm_pool_size="28GB",
+        CUDA_VISIBLE_DEVICES="1,2,3,4,5,6,7",  # let 0 be used by benchmark process
```

Comment:
Could we parameterize this, like Vibhu's (and also my code does) using the DASK_NUM_WORKERS environment variable?  i.e.

```
    n_devices = os.getenv('DASK_NUM_WORKERS', 4)
    n_devices = int(n_devices)
    visible_devices = ','.join([str(i) for i in range(1, n_devices+1)])
    start_dask_cluster(protocol='ucx', rmm_pool_size='28gb', CUDA_VISIBLE_DEVICES=visible_devices)
```

Reply:
addressed this in [a61cde7](https://github.com/rapidsai/cugraph/pull/3087/commits/a61cde71550a2cbfcaaaa0fc5dfa024072b6ae90)